main-app/app/scripts/game/controllers/game-controller.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+    'use strict';
+
+    angular.module('Tombola.Games.Bingo90.Game').
+        controller('game',
+        ['$scope', '$state', '$interval', 'GameServerProxy', 'RequestShaper', 'UserDetails', 'Ticket', 'CallHandler', 'GameState',
+            function ($scope, $state, $interval, proxy, req, userValue, ticket, callHandler, gameState) {
+                var me = this,
+                    timeBetweenCalls = 100;
+
+                $scope.currentCall = "";
+                $scope.ticket = ticket;
+                $scope.toGo = 5;
+                $scope.callNumber = 0;
+                $scope.currentStage = gameState.currentStage;
+
+                me.nextCall = function () {
+                    $scope.callNumber += 1;
+                    return $scope.callNumber;
+                };
+
+                me.markTicket = function (lastCall) {
+                    ticket.checkMark(lastCall);
+                };
+
+                me.getCall = function () {
+                    proxy.apiCall(req.makeCallRequest(userValue.data.token, me.nextCall())).then(
+                        function (data) {
+                            $scope.currentCall = callHandler.addNewCall(data);
+                            me.markTicket(me.formatSingleDigitCall(data.payload.call));
+                            $scope.toGo = ticket.calcToGo($scope.currentStage);
+                        },
+                        function (data) {
+                            $interval.cancel(callInterval);
+                        }
+                    );
+                };
+
+                var callInterval;
+                me.beginCalls = function () {
+                    callInterval = $interval(me.getCall, timeBetweenCalls, 90);
+                };
+
+                me.beginCalls();
+
+                $scope.logout = function () {
```

Comment:
As per `apiCall` - there is too much code in the controller.
The controller should pass off a simple call to the next layer down - Controllers are just a conduit between the UI (via the scope) and the mode (in the form of services).